src/main/java/bossmonster/Main.java

```diff
@@ -1,7 +1,221 @@
 package bossmonster;
 
+import java.util.Scanner;
+
 public class Main {
+
+    static Scanner sc = new Scanner(System.in);
+
+    static final int 물리_공격 = 1;
+    static final int 마법_공격 = 2;
+
+    static int 보스몬스터_최대_HP;
+    static int 보스몬스터_현재_HP;
+
+    static String 플레이어_이름;
+    static int 플레이어_최대_HP;
+    static int 플레이어_최대_MP;
+    static int 플레이어_HP;
+    static int 플레이어_MP;
+
+    static final String 보스_일반_얼굴 = "   ^-^" + "\n"
+            + " / 0 0 \\" + "\n"
+            + "(   \"   )" + "\n"
+            + " \\  -  /" + "\n"
+            + "  - ^ -";
+    static final String 보스_아픈_얼굴 = "   ^-^" + "\n"
+            + " / x x \\" + "\n"
+            + "(   \"\\  )" + "\n"
+            + " \\  ^  /" + "\n"
+            + "  - ^ -";
+    static final String 보스_이긴_얼굴 = "   ^-^" + "\n"
+            + " / ^ ^ \\" + "\n"
+            + "(   \"   )" + "\n"
+            + " \\  3  /" + "\n"
+            + "  - ^ -";
+
+
+    static int 반복_회수 = 0;
+
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        보스_몬스터_정보_입력();
+        플레이어_정보_입력();
+        게임_시작();
+        게임_결과();
+    }
+
+    static void 보스_몬스터_정보_입력() {
+        try {
+            System.out.println("보스 몬스터의 HP를 입력해주세요.");
+            보스몬스터_최대_HP = sc.nextInt();
+            보스몬스터_현재_HP = 보스몬스터_최대_HP;
+            sc.nextLine();
+            if (보스몬스터_최대_HP < 100 || 보스몬스터_최대_HP > 300) {
```

Comment:
개인적인 취향이지만 이런 식으로 바꾸면 `100 <= hp <= 300`처럼 읽을 수 있어 편할 수도 있을거 같네요~
```suggestion
            if (!(100 <= 보스몬스터_최대_HP && 보스몬스터_최대_HP <= 300)) {
```